index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>mobile-project</title>
+</head>
+<body>
+  <header>
+  <nav class="flex-nav">
+    <div>
+    <img src="images/MyLogo.png" alt="logo">
+    </div>
+    <div>
+    <img src="images/Portoflio.png" alt="Portoflio" class="right">
+    <img src="images/About.png" alt="About" class="right">
+    <img src="images/Contact.png" alt="Contact" class="right">
+  </div>
+  </nav>
+</header>
+  <main class="flex-main">
+    <section class="flex-center">
+   <h1>I’m Alexandra<br>
+    Glad to see <br> you!</h1>
+    <p class="">I’m a software developer! I can help you build a product , feature or we
+      bsite Look through some of my work and experience! If you like wha
+      t you see and have a project you need coded, don’t 
+      hestiate to contact me.</p>
+     
+    </section>
+      <section class = "flex-even">
+        <div><p>LETS CONNECT</p></div>
+        <div>
+        <img src="images/twitter.png" alt="twitter">
+        <img src="images/Vector.png" alt="Vector">
+        <img src="images/medium.svg" alt="medium">
+        <img src="images/github.png" alt="github">
+        <img src="images/angellist.png" alt="angellist">
+      </div>
+      </section>
+  </main> 
```

Comment:
- [ ] The section should have the proper background image (decorations are always background images, not `img` tags).
The background image you have here differs from the one used in Figma. The image placement has to do with the style placed on it.
_See the images below_
**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/120888752-f0ced400-c5ae-11eb-92a1-5418477c1c1f.png) | ![image](https://user-images.githubusercontent.com/49621279/120888796-2b387100-c5af-11eb-9e63-0b054e0e8e7d.png)

Reply:
These issues raised by the previous TSE still persist. Please make sure you made all the requested changes